athlinks-ontology.ttl

```diff
@@ -6,542 +6,680 @@
+# 
+# http://adp.athlinks.com/schema/AthleticEventProduct
+
+:AthleticEventProduct a owl:Class ;
+	rdfs:subClassOf schema:Product ;
+	rdfs:comment "An product offered in the context of an athletic event."@en ;
+	rdfs:label "Athletic event product"@en .
+# 
+# http://adp.athlinks.com/schema/AthleticEventRole
+
+:AthleticEventRole a owl:Class ;
+	rdfs:subClassOf schema:Role ;
+	rdfs:comment "An abstract role that encompasses all roles around athletic events."@en ;
+	rdfs:label "Athletic event role"@en .
+# 
+# http://adp.athlinks.com/schema/AthleticEventSeries
+
+:AthleticEventSeries a owl:Class ;
+	rdfs:subClassOf :AthleticEvent ;
+	rdfs:comment "A collection of athletic events, usually over the course of a single calendar year, that comprise a thematic sequence. Compare to an athletic event brand."@en ;
+	rdfs:label "Athletic event series"@en .
+# 
+# http://adp.athlinks.com/schema/AthleticRace
+
+:AthleticRace a owl:Class ;
+	rdfs:subClassOf :AthleticEvent ;
+	rdfs:comment "An athletic event where multiple people, participating either as individuals or in teams, compete over a closed course in one or more athletic disciplines."@en ;
+	rdfs:label "Athletic race"@en .
+# 
+# http://adp.athlinks.com/schema/BikeAction
+
+:BikeAction a owl:Class ;
+	rdfs:subClassOf schema:ExerciseAction ;
+	rdfs:comment "Exercise by biking"@en ;
+	rdfs:label "Bike Action"@en .
+# 
+# http://adp.athlinks.com/schema/CheckpointTimingDevice
+
+:CheckpointTimingDevice a owl:Class ;
+	rdfs:subClassOf :AthleticEventRole ;
+	rdfs:comment "A timing controller and sensors when deployed at a given checkpoint on a course."@en ;
+	rdfs:label "Checkpoint timing device"@en .
+# 
+# http://adp.athlinks.com/schema/CrossCountryTeam
+
+:CrossCountryTeam a owl:Class ;
+	rdfs:subClassOf :AggregateTeam ;
+	rdfs:comment "An aggregate team in a cross-country style event."@en ;
+	rdfs:label "Cross-country team"@en .
+# 
+# http://adp.athlinks.com/schema/CyclingEvent
+
+:CyclingEvent a owl:Class .
+# 
+# http://adp.athlinks.com/schema/DuathlonEvent
+
+:DuathlonEvent a owl:Class .
+# 
+# http://adp.athlinks.com/schema/LeaderBoard
+
+:LeaderBoard a owl:Class ;
+	rdfs:subClassOf :AthleteResultList ;
+	rdfs:comment "A short result list comprised of the current leaders of an athletic event. Often, this list is limited to a particular cohort of participants."@en ;
+	rdfs:label "Leader board"@en .
+# 
+# http://adp.athlinks.com/schema/LegacyDataRecord
+
+:LegacyDataRecord a owl:Class ;
+	rdfs:subClassOf schema:Intangible ;
+	rdfs:comment "A legacy data record holds structured information from a legacy data system."@en ;
+	rdfs:label "Legacy data record"@en .
+# 
+# http://adp.athlinks.com/schema/LegacyDataRole
+
+:LegacyDataRole a owl:Class ;
+	rdfs:subClassOf schema:Role ;
+	rdfs:comment "A role played by a legacy data record in the context of some thing."@en ;
+	rdfs:label "Legacy data role"@en .
+# 
+# http://adp.athlinks.com/schema/MountainBikeEvent
+
+:MountainBikeEvent a owl:Class .
```

Comment:
Should this and the next few classes be subclasses of `:AthleticEventCategory`? If no, what is the ontological difference between `:MountainBikeEvent` and `:AquabikeEvent`? If yes, we should colocate them with the other former `:ActivityType`s.

Reply:
They are, the problem is protege shredded the definitions. The rest of the defs are later in the file.